jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/CpuUtilizationHelper.java

```diff
@@ -64,7 +64,12 @@ synchronized double getSystemCpuLoad() {
 
 		/* calculate using the most recent value in the history */
 		if ((latestTime.getTimestamp() - interimTime.getTimestamp()) >= MINIMUM_INTERVAL) {
-			cpuLoad = calculateCpuLoad(latestTime, interimTime);
+	            if (latestTime.getCpuLoad() == -1) {
```

Comment:
For consistency, please use tabs rather than spaces throughout all JCL and VM code

Reply:
Changes applied in commit ffdf65f8be493319c906b24c60b752782364cb35